FluentAssertions.Net45.Specs/AsyncFunctionExceptionAssertionSpecs.cs

```diff
@@ -183,6 +183,29 @@ public void When_async_method_throws_exception_expected_not_to_be_thrown_it_shou
             action.ShouldThrow<AssertFailedException>()
                 .WithMessage("Did not expect System.ArgumentException, but found one*");
         }
+
+        [TestMethod]
+        public void When_async_method_throws_exception_without_inner_exception_it_should_throw_AssertFailedException()

```

Comment:
@dennisdoomen My poor English causes this confusing. I mean the situation an async method throws an exception that doesn't have an inner exception. Can you suggest a better test case name?